src/components/Modal/ModalInput.js

```diff
@@ -0,0 +1,45 @@
+import styled from 'styled-components';
+
+import { theme } from './../styles/theme';
+import TimerInput from './../Input/TimerInput';
+import { Flex } from './../styles/styledElements';
+
+function ModalInput() {
+    return (
+        <SingleWrapper padding>
+            <h3>Time (minutes)</h3>
+            <SingleWrapper
+                as='form' //not sure whether it's necessary'
```

Comment:
you need this because `SingleWrapper` is a div and you want to wrap your inputs into a form element. You did so using the `as` prop from styled-components: https://styled-components.com/docs/api#as-polymorphic-prop

the `as` prop is common in the react community and you may find it in many component libraries.